.evergreen/orchestration/drivers_orchestration.py

```diff
@@ -194,7 +194,7 @@ def run(opts):
     version = opts.version
     cache_dir = DRIVERS_TOOLS / ".local/cache"
     cache_dir_str = cache_dir.as_posix()
-    default_args = f"--out {mdb_binaries_str} --cache-dir {cache_dir_str}"
+    default_args = f"--out {mdb_binaries_str} --cache-dir {cache_dir_str} --retries 3"
```

Comment:
Historically, we seem to have used `--retry 5` with curl, e.g.: https://github.com/mongodb-labs/drivers-evergreen-tools/blob/1513f4964048be964a598cf7255f14f1d34accb5/.evergreen/download-mongodb.sh#L76

Is there a particular reason why `3` is chosen here?

Reply:
No, let's do 5